docs/en/observability/apm-error-count-threshold-rule.asciidoc

```diff
@@ -0,0 +1,116 @@
+[[apm-error-count-threshold-rule]]
+= Error count threshold rule
+
+Alert when the number of errors in a service exceeds a defined threshold. Error count rules can be set at the
+environment level, service level, and error group level.
+
+[discrete]
+[[apm-error-count-threshold-rule-filters-conditions]]
+== Filters and conditions
+
+// TO DO: Should we explain what an error grouping key is?
+// I couldn't find a good explanation anywhere else...
```

Comment:
Yes. You can find information on how the error grouping key is generated in this thread: https://discuss.elastic.co/t/about-the-principle-of-error-grouping-key/283622

Note that the file Andrew refers to in his answer now lives here: https://github.com/elastic/apm-data/blob/main/model/modelprocessor/groupingkey.go.